views/dashboard.html

```diff
@@ -57,6 +57,8 @@ <h1>{{ privateSpaceUsage }}</h1>
     </div>
   </section>
   <div class="links text-right">
+        <div class="pulse pull-left"></div><a href="/requests" class="pull-left">Try Premium Data</a> &nbsp;
```

Comment:
@tanvirchahal it says premium data but redirect to data requests page.. these are different services